### PR TITLE
Filepath fix

### PR DESCRIPTION
**Pull request by InAJacket**
Renamed folders to use lowercase. Names with uppercase were causing issues on Linux, as Windows' filesystem is not case-sensitive, and Linux's is.

### CODE REVIEW

**Review by noah1510 — CHANGES_REQUESTED** [review]
scripts/Classes is still uppercase.
Just change it while you are at it

File: ui/TestUI.tscn
Comment (noah1510) [inline ui/TestUI.tscn:1]: should be renamed to test_ui

File: maps/Bridge.tscn
Comment (noah1510) [inline maps/Bridge.tscn:1]: should be named brigde

File: environment/materials/Grass.material
Comment (noah1510) [inline environment/materials/Grass.material:1]: should be named grass

File: effects/Red_Glow.tres
Comment (noah1510) [inline effects/Red_Glow.tres:1]: should be named red_glow

**Comment by InAJacket** [comment]
I was under the impression that files aren't the issue, only folders. You're right about classes though, I must have missed it.

**Comment by InAJacket** [comment]
Nvm, you're right. There's a separate style guide for filenames that we should probably be following too: https://docs.godotengine.org/en/stable/tutorials/best_practices/project_organization.html

**Comment by InAJacket** [comment]
Closing to fix merge conflict. I'll open a new PR in a minute.

**Pull request closed without merging**